client/app/scripts/superdesk-search/search.js

```diff
@@ -1020,6 +1026,62 @@
                             }, 0, false);
                         });
                     });
+
+                    metadata.fetchSubjectcodes().then(function () {
+                        scope.subjectcodes = metadata.values.subjectcodes;
+                        tags.initSelectedFacets().then(function (currentTags) {
```

Comment:
For better readability, a flat promise chain would be better than nesting `.then()` calls.